src/PSRule/Definitions/Expressions/LanguageExpressions.cs

```diff
@@ -220,6 +247,24 @@ private static bool AcceptsWith(string[] with)
             }
             return false;
         }
+
+        private static bool AcceptsRule(string[] rule)
+        {
+            if (rule == null || rule.Length == 0)
+                return true;
+
+            var comparer = RunspaceContext.CurrentThread.Pipeline.Baseline.GetTargetBinding().IgnoreCase
```

Comment:
If we use `PSRule.Definitions.ResourceIdEqualityComparer` we should also allow a fully qualified rule id to be used such as `.\Rule1`.

Reply:
Ah yes, I'll include that. Thanks mate.